providers/prefab/src/test/java/dev/openfeature/contrib/providers/prefab/PrefabProviderTest.java

```diff
@@ -0,0 +1,174 @@
+        client = OpenFeatureAPI.getInstance().getClient();
+    }
+
+    @AfterAll
+    static void shutdown() {
+        prefabProvider.shutdown();
+    }
+
+    @Test
+    void getBooleanEvaluation() {
+        assertEquals(true, prefabProvider.getBooleanEvaluation(FLAG_NAME, false, new ImmutableContext()).getValue());
+        assertEquals(true, client.getBooleanValue(FLAG_NAME, false));
+        assertEquals(false, prefabProvider.getBooleanEvaluation("non-existing", false, new ImmutableContext()).getValue());
+        assertEquals(false, client.getBooleanValue("non-existing", false));
+    }
+
+    @Test
+    void getStringEvaluation() {
+        assertEquals(VARIANT_FLAG_VALUE, prefabProvider.getStringEvaluation(VARIANT_FLAG_NAME, "",
+            new ImmutableContext()).getValue());
+        assertEquals(VARIANT_FLAG_VALUE, client.getStringValue(VARIANT_FLAG_NAME, ""));
+        assertEquals("fallback_str", prefabProvider.getStringEvaluation("non-existing",
+    "fallback_str", new ImmutableContext()).getValue());
+        assertEquals("fallback_str", client.getStringValue("non-existing", "fallback_str"));
+    }
+
+    @Test
+    void getObjectEvaluation() {
+        assertEquals(VARIANT_FLAG_VALUE, prefabProvider.getStringEvaluation(VARIANT_FLAG_NAME, "",
+                new ImmutableContext()).getValue());
+        assertEquals(new Value(VARIANT_FLAG_VALUE), client.getObjectValue(VARIANT_FLAG_NAME, new Value("")));
+        assertEquals(new Value("fallback_str"), prefabProvider.getObjectEvaluation("non-existing",
+                new Value("fallback_str"), new ImmutableContext()).getValue());
+        assertEquals(new Value("fallback_str"), client.getObjectValue("non-existing", new Value("fallback_str")));
+    }
+
+    @Test
+    void getIntegerEvaluation() {
+        MutableContext evaluationContext = new MutableContext();
+        assertEquals(INT_FLAG_VALUE, prefabProvider.getIntegerEvaluation(INT_FLAG_NAME, 1,
+            evaluationContext).getValue());
+        assertEquals(INT_FLAG_VALUE, client.getIntegerValue(INT_FLAG_NAME, 1));
+        assertEquals(1, client.getIntegerValue("non-existing", 1));
+
+        // non-number flag value
+        assertEquals(1, client.getIntegerValue(VARIANT_FLAG_NAME, 1));
+    }
+
+    @Test
+    void getDoubleEvaluation() {
+        MutableContext evaluationContext = new MutableContext();
+        assertEquals(DOUBLE_FLAG_VALUE, prefabProvider.getDoubleEvaluation(DOUBLE_FLAG_NAME, 1.1,
+            evaluationContext).getValue());
+        assertEquals(DOUBLE_FLAG_VALUE, client.getDoubleValue(DOUBLE_FLAG_NAME, 1.1));
+        assertEquals(1.1, client.getDoubleValue("non-existing", 1.1));
+
+        // non-number flag value
+        assertEquals(1.1, client.getDoubleValue(VARIANT_FLAG_NAME, 1.1));
+    }
+
+//    @Test
+    void getBooleanEvaluationByUser() {
+        MutableContext evaluationContext = new MutableContext();
+        evaluationContext.add("domain", "prefab.cloud");
```

Comment:
hi @jkebinger,
can you help on why this evaluation not returning true by the context domain ?
[flag config](https://github.com/open-feature/java-sdk-contrib/pull/915/files#diff-f37820eae2aaf5dbf312964f585489ffe90fcb83c2f89441b3e8de6013e2dce6R13):
> just_my_domain: { "feature_flag": "true", value: "new-version", criteria: { operator: PROP_IS_ONE_OF, property: "domain", values: [ "prefab.cloud", "example.com" ] } }

Reply:
ok I managed to solved it via the docs, replaced the deprecated _yaml_ config with _json_ config.
You can review this PR on your convenient time.
For Java 8, you may update Prefab Java client to version 8 if there are not too many specific Java>8 code sections.
thanks